client/src/app/tabs/getEditMenu.js

```diff
@@ -50,6 +77,28 @@ export function getAlignDistributeEntries({
   }];
 }
 
+
+export function getColorEntries({
+  setColor
+}) {
+  return [{
+    label: 'Set Color',
+    enabled: setColor,
+    submenu: COLORS.map(color => {
+      return {
+        label: `${color.title}`,
+        enabled: setColor,
+        action: 'setColor',
+        options: {
+          fill: color.fill,
+          stroke: color.stroke
+        },
+        icon: `app/resources/icons/${color.title.toLowerCase()}-circle.png`

```

Comment:
Proposal: Do not encode full icon paths here (in the client); rather just pass the name (with extension) and make the menu infrastructure retrieve it, relative to a search directory the app knows (i.e. resources).

I.e. in this case, `resources/icons/${...}-circle.png` could be enough.

Looking into the future, plug-ins could ship their own icons, too, via the `app-plugins` protocoll, i.e. `app-plugins://token-simulation/resources/some-icon.png`.

Cf. https://github.com/camunda/camunda-modeler/blob/develop/app/lib/plugins.js#L150.

Reply:
I updated the client to pass on `resources/icons/${...}-circle.png` and the menu builder to add the `app/` prefix to the path. Is this what you meant?

Reply:
Exactly. Not encoding the base path allows us to support multiple search paths in the future.